pkg/env/node_scan.go

```diff
@@ -0,0 +1,18 @@
+package env
+
+import "time"
+
+var (
+	nodeRescanIntervalDefault = 4 * time.Hour
+	// NodeRescanInterval will set the duration for when to scan nodes for vulnerabilities (NodeScanV2)
+	NodeRescanInterval = registerDurationSetting("ROX_NODE_RESCAN_INTERVAL", nodeRescanIntervalDefault)
```

Comment:
[nit] This setting can be unexported because its value is available through `GetNodeRescanInterval`

```suggestion
	nodeRescanInterval = registerDurationSetting("ROX_NODE_RESCAN_INTERVAL", nodeRescanIntervalDefault)
```

Reply:
Yep, it makes sense.